tagstudio/src/qt/ts_qt.py

```diff
@@ -525,6 +525,14 @@ def start(self) -> None:
         elif self.settings.value(SettingItems.START_LOAD_LAST, True, type=bool):
             lib = self.settings.value(SettingItems.LAST_LIBRARY)
 
+            # TODO: Remove this check if the library is no longer saved with files
+            if not (Path(lib) / TS_FOLDER_NAME).exists():
+                logging.error(
+                    f"[QT DRIVER] {TS_FOLDER_NAME} folder in {lib} does not exist."
+                )
+                self.settings.setValue(SettingItems.LAST_LIBRARY, "")
+                lib = None
+
```

Comment:
If `LAST_LIBRARY` is absent from the config while `START_LOAD_LAST` remains (a situation I encountered while testing) then this block soft crashes if it doesn't verify that `lib` received a value from the config.
```suggestion
            if lib:
                # TODO: Remove this check if the library is no longer saved with files
                if not (Path(lib) / TS_FOLDER_NAME).exists():
                    logging.error(
                        f"[QT DRIVER] {TS_FOLDER_NAME} folder in {lib} does not exist."
                    )
                    self.settings.setValue(SettingItems.LAST_LIBRARY, "")
                    lib = None
```

Reply:
Ahh yeah I didn't think about that case. change implemented.